docs/content/setup/getting_started.md

```diff
@@ -12,23 +12,24 @@ You will have to accept the TLS warning in your browser the first time the page
 
 ## Production setup
 
+<!-- prettier-ignore -->
```

Comment:
It breaks the info boxes otherwise…

Reply:
Then add a comment please

Reply:
I'll add something in the documentation about the documentation. I think that's a better place for it. Because otherwise this comment will be quite numerous…